### PR TITLE
feat: add tooltip, truncation and formatting

### PR DESCRIPTION
**Pull request by ghost**
fixes:
- https://app.clickup.com/t/9002019994/PRF-6318
- https://app.clickup.com/t/9002019994/PRF-6410
- https://app.clickup.com/t/9002019994/PRF-6207
- https://app.clickup.com/t/9002019994/PRF-6315
- https://app.clickup.com/t/9002019994/PRF-6260

### CODE REVIEW

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6207 On cysync in send txn the amount field gets changed into scinetific notation](https://app.clickup.com/t/9002019994/PRF-6207)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6315 On cysync add balance truncation and show balances on hover](https://app.clickup.com/t/9002019994/PRF-6315)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6318 On cysync if wallet name is having more characters then overflow can be seen](https://app.clickup.com/t/9002019994/PRF-6318)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6410 On cysync in send flow the Amount and network fees value in $ should be upto 4 decimal places](https://app.clickup.com/t/9002019994/PRF-6410)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6410 On cysync in send flow the Amount and network fees value in $ should be upto 4 decimal places](https://app.clickup.com/t/9002019994/PRF-6410)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6260 On cysync in send txn of bitcoin conversion error is appearing](https://app.clickup.com/t/9002019994/PRF-6260)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6207 On cysync in send txn the amount field gets changed into scinetific notation](https://app.clickup.com/t/9002019994/PRF-6207)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6260 On cysync in send txn of bitcoin conversion error is appearing](https://app.clickup.com/t/9002019994/PRF-6260)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6318 On cysync if wallet name is having more characters then overflow can be seen](https://app.clickup.com/t/9002019994/PRF-6318)

**Comment by rohanagarwal94** [comment]
Task linked: [PRF-6315 On cysync add balance truncation and show balances on hover](https://app.clickup.com/t/9002019994/PRF-6315)

File: packages/coin-support-utils/src/unit.ts [hunk 1]
```diff
@@ -1,84 +1,36 @@
...
-    return resp;
-  }
-
-  const res = `${leftDecimalStr}${'.'}${rightDecimalStr.substring(
-    0,
-    lastIndex + 1,
-  )}${expStr}`;
-
-  return res;
+  const fixed = parseFloat(number.toFixed(decimal)).toString();
```
Comment (irshadCypherock) [inline packages/coin-support-utils/src/unit.ts]: Any reason we can't do this:
```suggestion
  const fixed = number.toFixed(decimal);
```

File: packages/cysync-core/src/hooks/useTransactions.tsx [hunk 2]
```diff
@@ -64,8 +65,8 @@ export interface TransactionRowData {
   dateTime: string;
   date: string;
   dateHeader: string;
-  amount: number;
-  value: number;
+  amount: string;
+  value: string;
```
Comment (irshadCypherock) [inline packages/cysync-core/src/hooks/useTransactions.tsx]: If this is a string, lodash will use string sorting for this.

File: packages/ui/src/components/atoms/Tooltip/index.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import RCTooltip from 'rc-tooltip';
+import React from 'react';
+
+export { TooltipStyles } from './styles';
+
+interface TooltipProps {
```
Comment (irshadCypherock) [inline packages/ui/src/components/atoms/Tooltip/index.tsx]: Always export prop types

**Review by irshadCypherock — CHANGES_REQUESTED** [review]

**Review by irshadCypherock — APPROVED** [review]

**Pull request merged**